resources/metadata/cloudFoundryDeleteService.yaml

```diff
@@ -66,6 +66,16 @@ spec:
         mandatory: true
         aliases:
           - name: cloudFoundry/serviceInstance
+      - name: cfDeleteServiceKeys
+        type: bool
+        description: Parameter to force deletion of Cloud Foundry Service Keys
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        aliases:
+          - name: cloudFoundry/cfServiceKeys
```

Comment:
cloudFoundry/cfServiceKeys -> cloudFoundry/cfDeleteServiceKeys

Reply:
changed accordinlgy. thanks for the hint